### DISCUSSION TITLE
[v5] - Upgrade from preview to release

### DISCUSSION THREAD

**Discussion by sn0rcha** [adonisjs/core#2547] [Help] [answered]
Hey Guys,

I had a quick look at the documentation and couldn't find any instructions of what should be done to move from the preview -> release version.

Has anyone got a summary of what changes should be made to the package.json to move from preview -> release.

Sorry if it's in the docs and I missed it!

Cheers,
Ben

**Comment by dqgiang** [comment, marked as answer]
I think you meant this below.
It's not in the docs. It's in the release note.
https://docs.adonisjs.com/releases/out-of-preview
![image](https://user-images.githubusercontent.com/1506041/117238610-6d0eb400-ae57-11eb-9d44-44a685e299de.png)

**Reply by sn0rcha** [reply]
Oh wow sorry about that I went straight to the new website and docs! - My bad

**Reply by sn0rcha** [reply]
Just a note on this - I had to do the core first otherwise it threw a dependency error.

**Comment by pk4all** [comment]
Hi Team,
I am getting an error after Upgrade when I run build command for build

"Cannot complete the build process as there are typescript errors. Use "--ignore-ts-error" flag to ignore Typescript errors"

![image](https://user-images.githubusercontent.com/12910992/119618645-c0ee3680-be20-11eb-91ec-c3d723c30bf6.png)

**Reply by thetutlage** [reply]
There are valid TypeScript errors that you need to fix. Can you share your `tsconfig.json` file?
